stdplugins/quotly.py

```diff
@@ -19,26 +19,25 @@ async def _(event):
         quote = reply
     else:
         return
-    bot = "@QuotLyBot"
-    await event.edit("```Making a quote....```")
-
-    async with borg.conversation(bot) as bot_conv:
-        if True:  # lazy indentation workaround xD
+    username = "@QuotLyBot"
+    await event.edit(f"```Quoting this message...```")
```

Comment:
Codacy found an issue: [f-string is missing placeholders (F541)](https://app.codacy.com/manual/prono69/PepeBot/pullRequest?prid=6191017)